EPM/v1/_init.yaml

```diff
@@ -1,13 +1,42 @@
 instances:
   - sal_index: 1
     data_clients:
-      # TODO DM-44580 This needs to be replaced with a real configuration.
+      # Needed for BTS and TTS simulator instances.
       - client_class: SnmpDataClient
         config:
           host: localhost
           port: 161
           max_read_timeouts: 5
           device_name: TestDevice
           device_type: scheiderPm5xxx
+          snmp_community: public
+          poll_interval: 1.0
+  - sal_index: 301
+    data_clients:
+      - client_class: SnmpDataClient
+        config:
+          host: 10.18.3.21
```

Comment:
Internal network instead of controller network? How do you sure the named IP will be always the same?

Reply:
Very good question! I will check this with IT.